foreman/data_refinery_foreman/surveyor/sra.py

```diff
@@ -338,42 +343,50 @@ def _generate_experiment_and_samples(self, run_accession: str) -> None:
                 experiment_object.pubmed_id = metadata["pubmed_id"]
                 experiment_object.has_publication = True
             if "study_ena_first_public" in metadata:
-                experiment_object.source_first_published = parse_datetime(metadata["study_ena_first_public"])
+                experiment_object.source_first_published = parse_datetime(
+                    metadata["study_ena_first_public"])
             if "study_ena_last_update" in metadata:
-                experiment_object.source_last_modified = parse_datetime(metadata["study_ena_last_update"])
+                experiment_object.source_last_modified = parse_datetime(
+                    metadata["study_ena_last_update"])
 
             # Rare, but it happens.
             if not experiment_object.protocol_description:
-                experiment_object.protocol_description = metadata.get("library_construction_protocol", "Protocol was never provided.")
+                experiment_object.protocol_description = metadata.get(
+                    "library_construction_protocol", "Protocol was never provided.")
 
             experiment_object.save()
 
-            ##
+            #
             # Experiment Metadata
-            ##
+            #
```

Comment:
Two comments were in place as 'section indicators - if you don't want these, just remove the whole line rather than making it like 

```
#
# this
#
```